stdlib/3/concurrent/futures/thread.pyi

```diff
@@ -13,6 +13,7 @@ if sys.version_info >= (3, 9):
 _S = TypeVar("_S")
 
 class ThreadPoolExecutor(Executor):
+    _work_queue: Any
```

Comment:
I think we could use a better type, but I'm not sure what it should be

ThreadPoolExecutor assigns it `queue.SimpleQueue` so it could be that, 
but it only uses a few methods so maybe a protocol would be better?

```python
class _QueueLike(Protocol):
    def get(self, block: bool = ...) -> Any: ...
    def get_nowait(self) -> Any: ...
    def put(self, item: Any) -> None: ...
```

Or maybe just `queue.Queue`?

rel: https://github.com/python/cpython/blob/7cf0aad96d1d20f07d7f0e374885f327c2d5ff27/Lib/concurrent/futures/thread.py#L144
rel: https://github.com/python/cpython/blob/7cf0aad96d1d20f07d7f0e374885f327c2d5ff27/Lib/queue.py

Reply:
In 3.6 it's actually a `queue.Queue`; haven't checked when the change was made. I think we should just use the concrete type that's present at runtime; it doesn't really make sense to introduce a custom protocol for a private field.